protected/humhub/modules/admin/messages/ru/user.php

```diff
@@ -18,84 +18,95 @@
-  'Default content of the registration denial email' => 'Содержимое письма об отказе в регистрации по умолчанию',
-  'Default group can not be deleted!' => 'Группу по умолчанию удалить невозможно!',
-  'Default user idle timeout, auto-logout (in seconds, optional)' => 'Время бездействия пользователя до авто-выхода по умолчанию (в секундах, опционально)',
+  'Default content of the registration approval email' => 'Содержимое электронного письма с информацией об одобрении по умолчанию',
+  'Default content of the registration denial email' => 'Содержимое электронного письма с информацией об отклонении по умолчанию',
+  'Default group can not be deleted!' => 'Группа по умолчанию не может быть удалена!',
+  'Default user idle timeout, auto-logout (in seconds, optional)' => 'Время бездействия пользователя по умолчанию, после чего будет автоматический выход из системы (в секундах, необязательно)',
   'Default user profile visibility' => 'Кому виден профиль пользователя по умолчанию',
-  'Delete' => 'Удалить',
+  'Default' => 'По умолчанию',
   'Delete All' => 'Удалить все',
-  'Delete all contributions of this user' => 'Удалить все вклады этого пользователя',
+  'Delete all contributions of this user' => 'Удалить все материалы этого пользователя',
   'Delete invitation' => 'Удалить приглашение',
   'Delete invitation?' => 'Удалить приглашение?',
-  'Delete spaces which are owned by this user' => 'Удалить пространства, принадлежащие этому пользователю',
-  'Deleted' => 'Удалить',
-  'Deleted invitation' => 'Удалено приглашение',
-  'Deleted users' => 'Удаленные пользователи',
-  'Disable' => 'Выключить',
+  'Delete pending registrations?' => 'Удалить ожидающие регистрации?',
+  'Delete spaces which are owned by this user' => 'Удалить все сообщества, принадлежащие этому пользователю',
+  'Delete' => 'Удалить',
+  'Deleted invitation' => 'Удалённое приглашение',
+  'Deleted users' => 'Удалённые пользователи',
+  'Deleted' => 'Удалено',
+  'Disable' => 'Отключить',
+  'Disabled users' => 'Отключённые пользователи',
   'Disabled' => 'Отключён',
-  'Disabled users' => 'Отключенные пользователи',
-  'Do not change placeholders like {displayName} if you want them to be automatically filled by the system. To reset the email content fields with the system default, leave them empty.' => 'Не меняйте условное название (placeholders), такие как {displayName}, если вы хотите, чтобы они автоматически заполнялись системой. Чтобы сбросить поля содержимого электронной почты до системных значений по умолчанию, оставьте их пустыми.',
-  'Edit' => 'Изменить',
-  'Edit category' => 'Изменить категорию',
+  'Do not change placeholders like {displayName} if you want them to be automatically filled by the system. To reset the email content fields with the system default, leave them empty.' => 'Пожалуйста, не меняйте заполнители типа {DisplayName}, если вы хотите, чтобы они автоматически заполнялись системой. Чтобы вернуть значения по умолчанию, оставьте поля пустыми.',
```

Comment:
My version seems more understandable to people who are not IT savvy.

Reply:
Ok, maybe yes, but I think anybody can understand what mean by words "заполнители" or "маски" because then we have the object `{displayName}` of the context.
But the main fix here is that we should not change `{displayName}` to `{DisplayName}`.
It seems it doesn't matter here, because the placeholder is not used as var, but in most cases it is used as:

```php
Yii::t('UserModule.notification', '{displayName} mentioned you in {contentTitle}.', [
  'displayName' => Html::tag('strong', Html::encode($this->originator->displayName)),
  'contentTitle' => $this->getContentInfo($this->source),
])
```

Reply:
@hide-me Ok, then if you prefer to use `заполнители` for the `placeholders`, could you please update this in all other places e.g. [here](https://github.com/humhub/humhub/pull/7326/files#diff-7f32b3464e06c9e51441808d3ba23ed8977200c2f474fe6a8d26aea685bce71fR118)?

file `protected/humhub/modules/admin/messages/ru/settings.php`:
```
'Use %url% as placeholder for URL. Format needs to be JSON. (e.g. http://www.youtube.com/oembed?url=%url%&format=json)'
=>
'Используйте плейсхолдер %url% вместо URL. Строка должна быть в JSON формате. (например: https://www.youtube.com/oembed?url=%url%&format=json)',
```

Reply:
Sure, thank you for noticing me about it!